conda.rb

```diff
@@ -7,14 +7,50 @@
 class Conda
   include Singleton
 
-  attr_reader :main
-
   def initialize
-    @main = Conda::Channel.new("pkgs/main", "repo.anaconda.com")
+    @channels = {
+      "main" => Conda::Channel.new("main", "repo.anaconda.com/pkgs"),
+      "conda-forge" => Conda::Channel.new("conda-forge", "conda.anaconda.org"),
+    }
+  end
+
+  def packages(channel)
+    raise Sinatra::NotFound unless @channels.key?(channel)
+
+    @channels[channel].packages
+  end
+
+  def all_packages
+    all_packages = {}
+    @channels.each_value do |channel|
+      channel.only_one_version_packages.each do |package_name, package|
```

Comment:
Can we just do the duduping at load time so all we have to hit is @packages ever?